### DISCUSSION TITLE
播放高画质视频会停止播放

### DISCUSSION THREAD

**Discussion by JiayanJohnnyChu** [ywmoyue/biliuwp-lite#579] [bug]
### Bug 描述

播放高画质视频（如4K或杜比视界）时，在播放数分钟后会停止播放并显示缓冲中，但会一直保持在这个状态无法继续播放，且此时通过任务管理器发现并没有实际的数据传输。（网络没有问题，任务管理器显示正常缓冲速度可达500mbps以上）
同时对于高画质视频，跳转到数分钟以后的位置也会一直显示缓冲中，任务管理器显示会有一个数百mbps的传输，其后便不再有网络活动。

### 操作步骤

```plain text
1. 打开应用并打开高画质视频（如当我来到世界尽头 https://b23.tv/BV1Vx4y1a7VB）。
2. 全屏播放该视频。
3. 数分钟后停止播放，并显示缓冲中。
4. 等待一段时间仍无法继续播放，暂停后再次播放也无法继续。
```

### 期望结果

正常播放高画质视频并能正常进行跳转

### 日志

[Untitled.txt](https://github.com/ywmoyue/biliuwp-lite/files/14967199/Untitled.txt)

### 应用来源

Github/其他Github镜像站

### 系统版本

Windows 11 23H2

### 设备类型

平板

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by JiayanJohnnyChu** [comment]
[Untitled.txt](https://github.com/ywmoyue/biliuwp-lite/files/14967256/Untitled.txt)
更详细的日志

**Comment by ywmoyue** [comment]
https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.4 试试4.7.4设置播放器类型为FFmpegInterop @JiayanJohnnyChu
